matplotlibrc.template

```diff
@@ -294,6 +294,7 @@
 #axes.grid.axis      : both    ## which axis the grid should apply to
 #axes.grid.which     : major   ## gridlines at major, minor or both ticks
 #axes.titlesize      : large   ## fontsize of the axes title
+#axes.titlelocation  : center  ## alignment of the title
 #axes.titleweight    : normal  ## font weight of title
```

Comment:
Maybe a very minor point here: I wouldn't put the location *in between* two parameters which set some font attributes. Maybe rather put it above `size` or below `weight`? Also the comment could mention the three possible values of this parameter.

Reply:
I think that's a valid point, I think above size would be a good location